src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -25,90 +25,99 @@ class AttributeComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
   "AttributeComponent" should "insert string attribute" in withEmptyTestDatabase {
     val testAttribute = AttributeString("test")
     runAndWait(workspaceQuery.save(workspace))
-    val numRows = workspaceAttributeQuery.insertAttributeRecords(workspaceId, "test", testAttribute, workspaceId).map(x => runAndWait(x))
+    val numRows = workspaceAttributeQuery.insertAttributeRecords(workspaceId, defaultAttributeName("test"), testAttribute, workspaceId).map(x => runAndWait(x))
```

Comment:
`defaultAttributeName` means set the attribute namespace to `default`.  Many tests had to be updated with this boilerplate.  Thanks to Biden for helping with this gruntwork.